src/main/java/seedu/address/model/attendance/exceptions/DuplicateAttendanceException.java

```diff
@@ -0,0 +1,12 @@
+package seedu.address.model.attendance.exceptions;
+
+/**
+ * Signals that the operation will result in duplicate Attendance
+ * (Attendance are considered duplicates if they have the same
+ * identity).
+ */
+public class DuplicateAttendanceException extends RuntimeException {
+    public DuplicateAttendanceException() {
+        super("Operation would result in duplicate attendance sessions");
```

Comment:
full stop? :x

Reply:
Sure thanks, although it is copied from the DuplicatePersonException